gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -38,7 +38,10 @@ def open(vm_disk)
     (Get-Item "#{@virtual_disk}").length
 STAT_EOL
     file_size, stderr = @parser.parse_single_powershell_value(run_correct_powershell(stat_script))
-    raise "Unable to obtain virtual disk size for #{vm_disk}" if stderr.include?("At line:")
+    if @network && stderr.include?("RegisterTaskDefinition")
+      raise "Unable to obtain virtual disk size for #{vm_disk}. Check Hyper-V Host Domain Credentials"
+    end
+    raise "Unable to obtain virtual disk size for #{vm_disk}" unless stderr.empty?
```

Comment:
The logic for raising an error seems less constrained than the previous version.
Before "At line:" had to be detected in the stderr stream. Now, it will raise an error if anything is in the stderr stream.
Are there any potential issues with that?